systests/jaxrs/src/test/java/org/apache/cxf/systest/jaxrs/description/SwaggerUiConfigurationQueryConfigTest.java

```diff
@@ -57,7 +59,7 @@ protected org.apache.cxf.endpoint.Server createServer(Bus bus) throws Exception
             sf.setProvider(new JacksonJsonProvider());
             final Swagger2Feature feature = new Swagger2Feature();
             feature.setRunAsFilter(false);
-            feature.setSwaggerUiConfig(new SwaggerUiConfig().url("/swagger.json").queryConfigEnabled(true));
```

Comment:
I removed the extra call since it should be set automatically. Now, with this removed, there would not be any relevant difference between this and `SwaggerUiConfigurationTest`, so perhaps this test would also be removed or some other difference should be introduced.

Reply:
We could test the behavior when the `queryConfigEnabled` is set to `false` explicitly fe.